handler/oauth2/flow_refresh.go

```diff
@@ -64,7 +64,14 @@ func (c *RefreshTokenGrantHandler) HandleTokenEndpointRequest(ctx context.Contex
 	refresh := request.GetRequestForm().Get("refresh_token")
 	signature := c.RefreshTokenStrategy.RefreshTokenSignature(refresh)
 	originalRequest, err := c.TokenRevocationStorage.GetRefreshTokenSession(ctx, signature, request.GetSession())
-	if errors.Is(err, fosite.ErrNotFound) {
+	if errors.Is(err, fosite.ErrInactiveToken) {
```

Comment:
If I understand correctly, the idea is that the storage would return a new error code `ErrInactiveToken` when a refresh token was used or revoked - am I correct with that assumption?

Reply:
Yes, exactly, same way it does for auth codes.

Reply:
Nice makes sense!